starport/pkg/plushhelpers/cast.go

```diff
@@ -8,8 +8,15 @@ import (
 // CastArgs returns the line of code to cast a value received from CLI of type string into its datatype
 // Don't forget to import github.com/spf13/cast in templates
 // TODO: Refactor this method to depend on datatypeName instead of datatype to make this method more maintainable if we add new types in the future
-func CastArgs(datatype string, argIndex int) string {
-	return fmt.Sprintf("cast.To%sE(args[%d])", strings.Title(datatype), argIndex)
+func CastArgs(datatypeName, datatype string, argIndex int) string {
+	if datatype == datatypeString {
+		return fmt.Sprintf("%s := args[%d]", datatypeName, argIndex)
+	}
+
+	return fmt.Sprintf(`%s, err := cast.To%sE(args[%d])
```

Comment:
Maybe we could also improve this one in this PR as well.

Currently we consider `To<TypeName>E` method exists for every types in `github.com/spf13/cast` but it will no longer be the case in the feature when we will add `Coin` type for example

We should use a switch case like inside the other methods of the same package